starters/pages-starter/components/navigation/HeaderNav.tsx

```diff
@@ -1,4 +1,4 @@
-import { Link } from "@/components/navigation/Link"
+import Link from "next/link"
```

Comment:
I think the changes here also need to be reverted to go back to the custom Link component.